examples/logger.py

```diff
@@ -133,6 +138,25 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         logger = logging.getLogger()
         logger.setLevel(logging.DEBUG)
         logger.addHandler(self.handler)
+        self.loggerFrame.levelBox.addItems(["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"])
+        self.loggerFrame.levelBox.textActivated.connect(self.setLevel)
+
+    @pyqtSlot(str)
+    def setLevel(self, text: str):
+        """Responds to the setLevelBox widget and changes the handler's level.
+        
+        Args:
+            text: Selected level in the level select box.
```

Comment:
Please add explanation about the input validity. For example:
`It should be one of DEBUG, INFO, WARNING, ERROR and CRITICAL (case-sensitive). Any other input is ignored.`